tests/unit/btree_stress_test.c

```diff
@@ -202,36 +223,26 @@ CTEST2(btree_stress, test_random_inserts_concurrent)
    uint64 root_addr = btree_create(
       (cache *)&data->cc, &data->dbtree_cfg, &mini, PAGE_TYPE_MEMTABLE);
 
-   platform_heap_id      hid     = platform_get_heap_id();
-   insert_thread_params *params  = TYPED_ARRAY_ZALLOC(hid, params, nthreads);
-   platform_thread      *threads = TYPED_ARRAY_ZALLOC(hid, threads, nthreads);
+   platform_heap_id      hid    = platform_get_heap_id();
+   insert_thread_params *params = TYPED_ARRAY_ZALLOC(hid, params, nthreads);
 
-   for (uint64 i = 0; i < nthreads; i++) {
-      params[i].cc        = (cache *)&data->cc;
-      params[i].cfg       = &data->dbtree_cfg;
-      params[i].hid       = data->hid;
-      params[i].scratch   = TYPED_MALLOC(data->hid, params[i].scratch);
-      params[i].mini      = &mini;
-      params[i].root_addr = root_addr;
-      params[i].start     = i * (nkvs / nthreads);
-      params[i].end = i < nthreads - 1 ? (i + 1) * (nkvs / nthreads) : nkvs;
-   }
+   load_thread_params(params,
```

Comment:
Modularized this out so it can be used in the `btree_print_diags()` test-case, which also has to do the same sort of thing to insert a bunch of data.